control_panel_api/services.py

```diff
@@ -163,3 +166,16 @@ def bucket_delete(name):
     """Simple facade function for view to call on s3bucket delete
     Note we do not destroy the actual data, just the policies"""
     delete_bucket_policies(name)
+
+
+def apps3bucket_delete(apps3bucket):
+    """On delete we detach the corresponding s3 read/write policy from app role
```

Comment:
This comment only describes _what_ the function does, it doesn't explain _why_

Reply:
Yep, also removed other ones.